src/client/client.js

```diff
@@ -1,5 +1,6 @@
 import {render} from 'react-dom';
 import React from 'react';
+import { AppContainer } from 'react-hot-loader';
```

Comment:
No lint error there. We're not using that rule, evidently. Perhaps we should?

Which is your preference: http://eslint.org/docs/rules/object-curly-spacing? 'never'?